### ISSUE TITLE
Structures generated on beaches float in the air

### ISSUE THREAD

**Issue by VisMGA** [Cristelknight/Expanded-Ecosphere#77]
![image](https://github.com/Cristelknight999/Expanded-Ecosphere/assets/35517046/d5c16f09-8287-49d8-84d0-68a9656bd9e4)
This image was produced in an instance with only WWEE, Crystel Lib, and Fabric API. Behavior found in 1.20.2 & 1.20.1. Others not tested.

Without other mods this is a rare edge case where a structure like a village has to generate nearby and extend onto the beach (only structures naturally spawning on beaches have rules that set their height, other than ruined portals which are otherworldly to begin with and generate with a large chunk of netherrack ground). Mods that add structures to beaches are a problem though.

It seems as though they are trying to match the height value of where the land would have been had it not been flattened.

**Comment by Cristelknight** [comment]
Yes exactly. And there is no easy way to fix this. When we find a way to change it easily would be great :)
